utils/subfiling_vfd/CMakeLists.txt

```diff
@@ -1,10 +1,20 @@
 cmake_minimum_required (VERSION 3.10)
 project (HDF5_UTILS_SUBFILINGVFD C)
 
-configure_file (${HDF5_UTILS_SUBFILINGVFD_SOURCE_DIR}/h5fuse.sh.in ${HDF5_BINARY_DIR}/h5fuse.sh @ONLY)
+configure_file (${HDF5_UTILS_SUBFILINGVFD_SOURCE_DIR}/h5fuse.sh.in ${HDF5_UTILS_SUBFILINGVFD_BINARY_DIR}/h5fuse.sh @ONLY)
```

Comment:
Optional: Note that HDF5_UTILS_SUBFILINGVFD_SOURCE(BINARY)_DIR could also be PROJECT_SOURCE(BINARY)_DIR. I do like the specific name though.